src/tools/pub/automated-publishing.md

```diff
@@ -116,6 +116,9 @@ jobs:
 
 Make sure to match the pattern in `on.push.tags` with the _tag pattern_
 specified on pub.dev. Otherwise, the GitHub Action workflow won’t work.
+If publishing multiple packages from the same repository, use a per-package
+_tag pattern_ like `my_package_name-v{{version}}` can create a workflow file
```

Comment:
I'm assuming the tag pattern and the "workflow file for each package" are two separate ideas, but this text seems to combine them. Did you intend to separate these?

Reply:
You'll need a workflow file for each tag-pattern.

There many ways to configure it, but that's the simplest one.

Reply:
You'll configure a separate tag-pattern for each package on pub.dev, then create a workflow file for each package.
The workflow files will need to have a trigger pattern that aligns with the tag pattern for the package referenced in working-directory

Reply:
Thanks for the extra context!